teaser/data/output/teaserjson_output.py

```diff
@@ -329,10 +289,12 @@ def set_basic_data(wall_out, element):
         wall_out["outer_radiation"] = element.outer_radiation
         wall_out["outer_convection"] = element.outer_convection
 
-    elif type(element).__name__ == 'InnerWall' or \
-            type(element).__name__ == 'Ceiling' or \
-            type(element).__name__ == 'Floor' or \
-            type(element).__name__ == 'GroundFloor':
+    elif (
+        type(element).__name__ == "InnerWall"
+        or type(element).__name__ == "Ceiling"
+        or type(element).__name__ == "Floor"
+        or type(element).__name__ == "GroundFloor"
+    ):
 
         wall_out["year_of_construction"] = element.year_of_construction
```

Comment:
not regarding this PR, but shouldn't we take:
`        wall_out["year_of_construction"] = element.year_of_construction
        wall_out["year_of_retrofit"] = element.year_of_retrofit
        wall_out["construction_type"] = element.construction_type

        wall_out["area"] = element.area
        wall_out["tilt"] = element.tilt
        wall_out["orientation"] = element.orientation

        wall_out["inner_radiation"] = element.inner_radiation
        wall_out["inner_convection"] = element.inner_convection`
out of the if statement? This would save us a view lines and reduce redundancy

Reply:
nice catch :) i changed all modules (input/output) accordingly: https://github.com/RWTH-EBC/TEASER/commit/ca72af9d09915f75cfca8878884bd0181901878f